spec/compiler/interpreter/spec_helper.cr

```diff
@@ -3,24 +3,45 @@ require "../spec_helper"
 require "compiler/crystal/interpreter/*"
 
 def interpret(code, *, prelude = "primitives")
-  context, value = interpret_with_context(code, prelude: prelude)
-  value.value
+  if prelude == "primitives"
+    context, value = interpret_with_context(code, prelude)
+    value.value
+  else
+    interpret_in_separate_process(code, prelude)
+  end
 end
 
-def interpret_with_context(code, *, prelude = "primitives")
+def interpret_with_context(code, prelude = "primitives")
```

Comment:
Wouldn't it make sense to make this one private to avoid shooting ourselves in the foot?

Reply:
We can't make this helper private, because it's used in specs when it needs access to the context, instead of just the value.
But I suppose we can remove the `prelude` parameter to fix it to `"primitives"`.